README.md

```diff
@@ -9,5 +9,23 @@ The front-end component of `slugline`, math**NEWS**'s best and only React/Django
 - Grab a copy of the server [here](https://github.com/UWmathNEWS/slugline-api). 
 - By default, the server runs at port 8000, and the `proxy` setting in `package.json` is designed to redirect requests from the dev server to there. If you change the server port, change the port in `package.json` as well.
 
-## Contributing
-Submit a pull request or something.
+## License
+Except for files which specify otherwise, this application is licensed under the [GNU Affero General Public License](https://www.gnu.org/licenses/agpl-3.0.en.html).
+
+```
+<one line to give the program's name and a brief idea of what it does.>
+Copyright (C) 2020  <name of author>
```

Comment:
Who are the copyright holders?

Reply:
Copyright is held by individual contributors for only the files they contributed to.